### PR TITLE
Closes #12. Implement From trait for Set and Map.

### LINKED ISSUE #12: impl From<Fst> trait to Fst::Map
It would allow users to build a `fst::Map` from a Mmap object.

### PR DESCRIPTION
**Pull request by fulmicoton**

### CODE REVIEW

File: src/map.rs [hunk 1]
```diff
@@ -2,6 +2,7 @@ use std::fmt;
 use std::iter::FromIterator;
 use std::io;
 use std::path::Path;
+use std::convert::From;
```
Comment (BurntSushi) [inline src/map.rs]: You should be able to remove this line, since `From` is in the prelude.

**Comment by BurntSushi** [comment]
One little nit, but this otherwise looks good! Thanks!

**Comment by fulmicoton** [comment]
I removed the imports.

**Comment by BurntSushi** [comment]
Awesome, thanks!

**Comment by BurntSushi** [comment]
I've released a new version to crates.io with this patch.

**Pull request merged**